siliconcompiler/utils/issue.py

```
@@ -115,91 +116,92 @@ def determine_copy(*keypath, in_require):
-    with task_class.runtime(SchedulerNode(chip, step, index), relpath=new_work_dir) as task_obj:
+    with task_class.runtime(SchedulerNode(project, step, index), relpath=new_work_dir) as task_obj:
         # Rewrite replay.sh
-        prev_quiet = chip.get('option', 'quiet', step=step, index=index)
-        chip.set('option', 'quiet', True, step=step, index=index)
+        prev_quiet = project.get('option', 'quiet', step=step, index=index)
+        project.set('option', 'quiet', True, step=step, index=index)
         try:
             # Rerun pre_process
             task_obj.pre_process()
         except Exception:
             pass
```

Ruff (0.13.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)